wfl/Controllers/FoodOrdersController.cs

```diff
@@ -42,5 +42,31 @@ public FoodOrdersController(IDBManager dbManager)
         {
             return Ok(await dbManager.GetEntityByIdAsync<FoodOrder>(id));
         }
+
+        [Route("{id}/discard")]
```

Comment:
How about making a more generic route that we can pass in a `Status` to? That way if we add additional statuses, we don't have to add new routes.